frontend/src/components/Review/ReviewHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from 'react';
+import Grid from '@material-ui/core/Grid';
+import styles from './Review.module.scss';
+import LabeledLinearProgress from '../utils/LabeledLinearProgress';
+import Button from '@material-ui/core/Button';
+import { RatingInfo } from '../../pages/LandlordPage';
+
+type Props = {
+  readonly numReviews: number;
+  readonly aveRatingInfo: RatingInfo[];
+};
+
+export default function PropertyInfo({ numReviews, aveRatingInfo }: Props): ReactElement {
+  return (
+    <div>
+      <div className={styles.headerContainer}>
+        <h2 className={styles.headerTitle}>Reviews ({numReviews})</h2>
+        <Button className={styles.button} variant="contained">
+          Leave a Review
+        </Button>
+      </div>
+
+      <div className={styles.detail}>
+        <Grid className={styles.infoContainer} container spacing={2} direction="row">
+          {aveRatingInfo.map(({ feature, rating }: RatingInfo, index) => (
```

Comment:
A very subjective thing but I prefer letting the type be inferred in this case because we already know aveRatingInfo is typed RatingInfo[]